go.mod

```diff
@@ -169,6 +169,7 @@ require (
 
 // Below are the long-lived replace of the Cosmos SDK
 replace (
+	cosmossdk.io/core => ./core
```

Comment:
Maybe best to split the PR to contain only the core changes.

Reply:
Yes but I'm not sure the steps, just to replace with newly added fields to make test happy.

Reply:
It is best to open a another PR with just the core and api changes. Then use a pseudo version here once that PR is merged. This makes the PR smaller and remove the need of those replace.